lms/djangoapps/grades/tests/test_grades.py

```diff
@@ -222,51 +228,134 @@ def create_location(self, block_type, block_id):
         )
 
     def test_score_chapter(self):
-        earned, possible = self.progress_summary.score_for_module(self.loc_a)
+        earned, possible = self.course_grade.score_for_module(self.loc_a)
```

Comment:
If all of these named locations exist for the sake of a single test, maybe rename them to reflect the test in which they are used.